test/intl402/Temporal/PlainYearMonth/from/reference-day-chinese.js

```diff
@@ -75,9 +71,9 @@ for (let [nonLeapMonthCode, month, referenceISODay] of months2022TestData) {
   );
 }
 
-// Years in which leap months exist according to ICU4C/ICU4X.
+// Years in which leap months exist according to
+// https://github.com/tc39/proposal-intl-era-monthcode/issues/60#issuecomment-3336312868
 const leapMonthsTestData = [
-  ["M01L", 1651, 2, 20],
   ["M02L", 2023, 3, 22],
   ["M03L", 1993, 4, 22],
   ["M04L", 2020, 5, 23],
```

Comment:
These might be replaced by a hardcoded table of reference years. But if we're deleting M01L for the time being, we should delete M11L (year 1517) as well.